src/index.html

```diff
@@ -6,9 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#" class="logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a href="#" class="link is-active">Apple</a>
+            </li>
+            <li class="nav__list-item">
+              <a href="#" class="link">Samsung</a>
+            </li>
+            <li class="nav__list-item">
+              <a href="#" class="link">Smartphones</a>
+            </li>
+            <li class="nav__list-item">
+              <a href="#"
+              data-qa="hover"
+              class="link">Laptops & Computers</a>
+            </li>
+            <li class="nav__list-item">
+              <a href="#" class="link">Gadgets</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__list-item">
              <a href="#" class="link">Smartphones</a>
            </li>
            
            <li class="nav__list-item">
              <a 
                href="#"
                data-qa="hover"
                class="link"
              >
                Laptops & Computers
              </a>
            </li>
            
            <li class="nav__list-item">
              <a href="#" class="link">Gadgets</a>
            </li>
```
Pay attention to the formatting